src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts

```diff
@@ -120,6 +121,12 @@ export const TERMINAL_TAB_ACTIVE_BORDER = registerColor('terminal.tab.activeBord
 	hcDark: TAB_ACTIVE_BORDER,
 	hcLight: TAB_ACTIVE_BORDER
 }, nls.localize('terminal.tab.activeBorder', 'Border on the side of the terminal tab in the panel. This defaults to tab.activeBorder.'));
+export const TERMINAL_INITIAL_HINT_FOREGROUND = registerColor('terminal.initialHint.foreground', {
+	dark: Color.fromHex('#ffffff56'),
+	light: Color.fromHex('#0007'),
```

Comment:
I see we don't use this `Color.fromHex` for other colors 🤔 . I'm wondering why 👀

Reply:
looks like that's not necessary, removing